fpga_interchange/fasm_generators/xc7.py

```diff
@@ -351,7 +351,7 @@ def handle_slice_ff(self):
             if cell_type.startswith("LD"):
                 self.add_cell_feature((tile_name, slice_site, "LATCH"))
 
-            if cell_type in ["FDRE", "FDCE"]:
+            if cell_type in ["FDRE"]:
```

Comment:
should probably be set for `FDSE` too, which has a synchronous set (it's not set for `FDCE` or `FDPE`, those have async clear/preload)

Reply:
Right, fixed